composer.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "automattic/crowdsignal-plugin",
+  "description": "Create and manage Crowdsignal polls and ratings in WordPress",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0-or-later",
+  "homepage": "https://crowdsignal.com/",
+  "authors": [
+    {
+      "name": "Automattic, Inc.",
+      "homepage": "https://automattic.com/"
+    }
+  ],
+  "support": {
+    "issues": "https://github.com/automattic/polldaddy-plugin/issues",
+    "source": "https://github.com/automattic/polldaddy-plugin"
+  },
+  "require": {
+    "composer/installers": "^1.0 || ^2.0",
+    "php": ">=5.6"
+  },
+  "require-dev": {
+    "wp-coding-standards/wpcs": "^3.0",
+    "automattic/vipwpcs": "^3.0",
+    "dealerdirect/phpcodesniffer-composer-installer": "^1.0"
+  },
+  "scripts": {
+    "phpcs": "phpcs",
+    "phpcbf": "phpcbf"
```

```suggestion
    "phpcs": "@php ./vendor/bin/phpcs",
    "phpcbf": "@php ./vendor/bin/phpcbf"
```

By using this approach, we ensure that the local instances of the PHPCS/PHPCBF tools are used, and not any globally installed version. We also add support for Windows users.